packages/react-components/react-skeleton/stories/Skeleton/SkeletonItemShape.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { Skeleton, SkeletonItem, makeStyles, tokens } from '@fluentui/react-components';
+
+const useStyles = makeStyles({
+  invertedWrapper: {
+    background: tokens.colorNeutralBackground1,
+    display: 'flex',
+    padding: tokens.spacingHorizontalXL,
+  },
+  row: {
+    display: 'grid',
+    gap: tokens.spacingHorizontalL,
+    gridTemplateColumns: '1fr 150px 1fr',
+  },
+});
+
+export const SkeletonItemShape = () => {
```

Comment:
I'd lean towards having these just be titled "Size" and "Shape"; I think the descriptions you added + the code makes it clear that the props go on `SkeletonItem` (similar to how "Row" is used as a title even though it's nested styles within Skeleton)

Reply:
Yep, I thought about it as well during implementation. Thanks, will update ❤️